spec/models/article_spec.rb

```diff
@@ -29,4 +29,39 @@
       end
     end
   end
+
+  context 'scopes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.